infrastructure/terraform/README.md

```diff
@@ -60,7 +42,45 @@ variable "doks_k8s_version" {
     cp bootstrapper.tfvars.sample bootstrapper.tfvars
     ```
 
-4. Open the `bootstrapper.tfvars` file and adjust settings according to your needs using a text editor of your choice (preferably with [HCL](https://github.com/hashicorp/hcl/blob/main/hclsyntax/spec.md) lint support).
+4. Open the `bootstrapper.tfvars` file and adjust settings according to your needs using a text editor of your choice (preferably with [HCL](https://github.com/hashicorp/hcl/blob/main/hclsyntax/spec.md) lint support). To see the default values, look at `variables.tf`.
+
+     **Important:** 
+    You will need to replace the `do_token` with your own Personal Access Token.
+
+    List Kubernetes versions that can be used with DigitalOcean clusters.
+
+    ```console
+    doctl k8s options versions
+    ```
+    ```bash
```

Comment:
@tiffanyfay Merge this block with the ```console``` block on top as this is just the output. Else, it appears as a multiline command that one can copy.